src/devices/CharacterLcd/Aip31068Lcd.cs

```diff
@@ -0,0 +1,167 @@
+            : base(new Size(16, 2), LcdInterface.CreateI2c(device, true))
+        {
+            _contrast = NormalizeContrast(contrast);
+            _iconDisplayEnabled = iconDisplayEnabled;
+            _boosterEnabled = boosterEnabled;
+
+            InitializeController();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Aip31068Lcd"/> class using an existing LCD interface.
+        /// </summary>
+        /// <param name="lcdInterface">The LCD interface used to communicate with the controller.</param>
+        /// <param name="contrast">Initial contrast value. Valid range: 0-63.</param>
+        /// <param name="iconDisplayEnabled">True to enable the icon display, false to disable it.</param>
+        /// <param name="boosterEnabled">True to enable the internal voltage booster, false to disable it.</param>
+        public Aip31068Lcd(LcdInterface lcdInterface, byte contrast = DefaultContrast, bool iconDisplayEnabled = false, bool boosterEnabled = true)
+            : base(new Size(16, 2), lcdInterface)
+        {
+            _contrast = NormalizeContrast(contrast);
+            _iconDisplayEnabled = iconDisplayEnabled;
+            _boosterEnabled = boosterEnabled;
+
+            InitializeController();
+        }
+
+        /// <summary>
+        /// Gets or sets the display contrast (0-63).
+        /// </summary>
+        public byte Contrast
+        {
+            get => _contrast;
+            set
+            {
+                byte normalized = NormalizeContrast(value);
+                if (_contrast == normalized)
+                {
+                    return;
+                }
+
+                _contrast = normalized;
+                UpdateContrastConfiguration();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the icon display is enabled.
+        /// </summary>
+        public bool IconDisplayEnabled
+        {
+            get => _iconDisplayEnabled;
+            set
+            {
+                if (_iconDisplayEnabled == value)
+                {
+                    return;
+                }
+
+                _iconDisplayEnabled = value;
+                UpdateContrastConfiguration();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the voltage booster is enabled.
+        /// </summary>
+        public bool BoosterEnabled
+        {
+            get => _boosterEnabled;
+            set
+            {
+                if (_boosterEnabled == value)
+                {
+                    return;
+                }
+
+                _boosterEnabled = value;
+                UpdateContrastConfiguration();
+            }
+        }
+
+        private void InitializeController()
+        {
+            EnterExtendedInstructionSet();
```

Comment:
can you point where in the documentation this can be found? Will make it easier for maintenance. Including the delay. Thanks!